LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -2365,6 +2365,91 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
 
                             break;
                         }
+                        case Model.ROG_STRIX_B550_I_GAMING: //NCT6798D
+                        {
+                            v.Add(new Voltage("Vcore", 0, 10, 10));
+                            v.Add(new Voltage("+5V", 1, 4, 1)); //Probably not updating properly
+                            v.Add(new Voltage("AVCC", 2, 10, 10));
+                            v.Add(new Voltage("+3.3V", 3, 34, 34));
+                            v.Add(new Voltage("+12V", 4, 11, 1)); //Probably not updating properly
+                            //v.Add(new Voltage("#Unused #5", 5, 0, 1, 0, true));
+                            //v.Add(new Voltage("#Unused #6", 6, 0, 1, 0, true));
+                            v.Add(new Voltage("3VSB", 7, 34, 34));
+                            v.Add(new Voltage("VBat", 8, 34, 34));
+                            v.Add(new Voltage("VTT", 9));
+                            //v.Add(new Voltage("#Unused #9", 9, 0, 1, 0, true));
+                            //v.Add(new Voltage("#Unused #10", 10, 0, 1, 0, true));
+                            //v.Add(new Voltage("#Unused #11", 11, 0, 1, 0, true));
+                            //v.Add(new Voltage("#Unused #12", 12, 0, 1, 0, true));
+                            //v.Add(new Voltage("#Unused #13", 13, 0, 1, 0, true));
+                            //v.Add(new Voltage("#Unused #14", 14, 0, 1, 0, true));
+
+                            t.Add(new Temperature("CPU", 1));
+                            t.Add(new Temperature("MotherBoard", 2));
```

Comment:
Motherboard

Reply:
I just wanted to keep the names according to AI Suite for easier understanding for users